### ISSUE TITLE
set-env issue

### ISSUE THREAD

**Issue by LanaManaseer90** [Malinskiy/action-android#32]
**I have these issues:**

- The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

- Unable to process command '::set-env name=ANDROID_HOME,::/home/runner/android-sdk' successfully.

**Please check the below screenshots for `the issue I faced` & the `.yml code`.**

<img width="510" alt="Screen Shot 2020-11-17 at 9 07 42 AM" src="https://user-images.githubusercontent.com/22792568/99357359-6fcf1a00-28b4-11eb-8d7c-b294be8fba2e.png">

<img width="1091" alt="Screen Shot 2020-11-17 at 9 07 21 AM" src="https://user-images.githubusercontent.com/22792568/99357361-7198dd80-28b4-11eb-90db-0c4ee847f7ef.png">

**Comment by YusukeSuzuki1213** [comment]
I have same issue.

Perhaps merging the following PRs will fix it.

https://github.com/Malinskiy/action-android/pull/31
https://github.com/Malinskiy/action-android/pull/30

According to [this information,](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/) the following statements are made.

> Action authors who are using the toolkit should update the @actions/core package to v1.2.6 or greater to get the updated addPath and exportVariable functions.

**Comment by LanaManaseer90** [comment]
@YusukeSuzuki1213 Thanks a lot for your comment, yes maybe if they merge these PRs that you mention it will fix the problem.

**Comment by LanaManaseer90** [comment]
@Malinskiy please advice :)

**Comment by hannesa2** [comment]
I merged all `build(deps): bump @actionsXXXXX` pull requests, made a new release 
https://github.com/hannesa2/action-android/releases/tag/0.0.7.6

used it

https://github.com/hannesa2/ioio/pull/12/checks?check_run_id=1424319375

but sadly the error still exists

**Comment by horie1024** [comment]
I have forked and published a workaround version.

Use it like this.

```yml
uses: horie1024/action-android/install-sdk@release/0.0.9
```

**Comment by hannesa2** [comment]
> I have forked and published a workaround version.
> 
> Use it like this.
> 
> ```yaml
> uses: horie1024/action-android/install-sdk@release/0.0.9
> ```

Sorry, it's not working on my side https://github.com/hannesa2/ioio/pull/15
<img width="963" alt="image" src="https://user-images.githubusercontent.com/3314607/99767384-3d672c00-2b03-11eb-9a5e-d520db9995e4.png">

**Comment by horie1024** [comment]
@hannesa2 
I'll check the problem.
Can I ask you to try this one instead?

```
uses: horie1024/action-android/install-sdk@release/0.0.8
```

**Comment by hannesa2** [comment]
I can confirm, It's working 👍  https://github.com/hannesa2/ioio/pull/15

`uses: horie1024/action-android/install-sdk@release/0.0.9`

**Comment by hannesa2** [comment]
@horie1024 I can't make issues on your fork, so I've to spam here ....sorry

I did the similar like you did, and to learn something, I want to ask you where is the magic ?
I made a pull request to my fork with your squashed changes, to see the diff. It would be nice, if you point me to the breaking change. https://github.com/hannesa2/action-android/pull/2 
Thank you

**Comment by horie1024** [comment]
@hannesa2 

No problem!
I commented on this question in your Pull Request. Please take a look.

https://github.com/hannesa2/action-android/pull/2#issuecomment-730953320

**Comment by LanaManaseer90** [comment]
@hannesa2 @horie1024 thanks a lot for your support, its working for me too 💯

**Comment by Malinskiy** [comment]
Fixed in 0.0.8, please use:
```yaml
- uses: malinskiy/action-android/emulator-run-cmd@release/0.0.8
- uses: malinskiy/action-android/install-sdk@release/0.0.8
```

Thanks, everyone for quick hotfixes!

**Comment by LanaManaseer90** [comment]
@Malinskiy thanks a lot 👍
